app/models/message.rb

```diff
@@ -0,0 +1,2 @@
+class Message < ApplicationRecord
+end
```

Comment:
It is normally not a good practice to leave your model files empty. You can consider adding custom scopes, default values, associations or validations in here for better control. For instance to validate the message and make sure it is not empty you can add the line below:  Kindly check [this link](https://rails.devcamp.com/trails/learn-ruby-on-rails-from-scratch/campsites/rails-models/guides/purpose-of-model-files-in-rails) for more information. :pray: 

           validates :message, presence: true, length: { in: 1..200 }

Reply:
Hi @jerryowusu, Thanks for your feedback, I added your suggested change:

![image](https://github.com/LarryIVC/hello-rails-back-end/assets/110146573/45a5a20c-ff1b-49eb-9d4d-906f2e8f139a)